content/blog/2021-08-11-unconference.md

```diff
@@ -0,0 +1,21 @@
++++
+title="Unconference 2021"
+[extra]
+author="Nordic-RSE"
++++
+
+# Unconference 2021
+Although the world situation prevented our plans to meet the community in person in Stockholm and we were all busy with other works, we did not want to give up meeting the community this year. Thus we decided for the unconference/birds of a feather format to keep organizational hassle low, but still give the community a place to meet, learn, and discuss. In that way, the community could decide on the topics of interest and create an event to their liking.
+The event was held on June 29th + 30th 2021 afternoons. Three speakers were invited beforehand to talk about their interesting work, 
+the rest of the event was free for the community to fill.
+* Shahnawaz Ahmed from the Wallenberg Centre for Quantum Technology talked about the journey of his Quantum Toolbox in Python (QuTiP) from small research project 
+to open source project developed by a community from all around the world.
+* Kristoffer Carlsson  from JuliaComputing introduced Julia for research software.
+* Anne Fouilloux from Nordic-RSE gave some insights into myths and misconceptions about research software development in academia that she encountered during her work
+as an RSE.
+The event had 69 registrations from 7 countries (FI,NO,SE,DK,UK,DE,NL). 
+Topics ranged from broad (Introduction to Julia, What is R/S/E?, ...) to specific ( Funding for software projects, Development frameworks, Software paper publishing,...) 
+with lots of Q&A and discussions. You can find schedule and notes [here](https://nordic-rse.org/events/2021-online-unconference/#schedule) and stay tuned 
+for more blogposts (https://nordic-rse.org/blog/) and recordings of the invited talks coming in the near future.
+Since this was the first event of this format, we learned a lot in the process. We definitely want to organize this kind of event again, but will try to bring a bit more order into the submission and decision of talks and discussions.
```

Comment:
```suggestion
Since this was the first event of this format, we learned a lot in the process. We definitely want to organize this kind of event again, but will try to bring a bit more order into the submission and decision of talks and discussions.
Another suggestion we got was to encourage last minute contributions by having more own contributions since it is easier to add the tenth suggestion to a list of nine suggestions rather than adding the first.
```

Reply:
It may not be clear from the diff but only the last sentence got added but nothing else changed.